k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -747,6 +749,18 @@ private void onDiscard() {
         }
     }
 
+    private void onHighPriority(MenuItem item) {
+        if (isHighPriority) {
+            item.setTitle(getString(R.string.priority_high));
+            setTitle(getString(R.string.compose_title_compose));
+            this.isHighPriority = false;
+        } else {
+            item.setTitle(getString(R.string.priority_normal));
+            setTitle(String.format("%s,%s", getTitle(), getString(R.string.priority_high)));
```

Comment:
I still don't like `setTitle` depending on `getTitle`. Just set it to another string resource